gravitee-apim-console-webui/src/components/notification/notification-edit-dialog/notification-edit-dialog.component.html

```diff
@@ -40,6 +40,16 @@ <h2 mat-dialog-title>Edit {{ notification.name }}</h2>
         </gio-form-slide-toggle>
       </div>
     </ng-container>
+    <ng-container *ngIf="isPortalNotification && isPrimaryOwner" >
+      <mat-form-field appearance="fill">
```

Comment:
Same here to be consistent with the others mat-form-fields styles maybe
```suggestion
      <mat-form-field>
```